src/main/java/seedu/moneygowhere/ui/GraphPanel.java

```diff
@@ -0,0 +1,62 @@
+package seedu.moneygowhere.ui;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.logging.Logger;
+
+import javafx.fxml.FXML;
+import javafx.scene.chart.CategoryAxis;
+import javafx.scene.chart.LineChart;
+import javafx.scene.chart.NumberAxis;
+import javafx.scene.chart.XYChart;
+import javafx.scene.control.Tooltip;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.StackPane;
+import seedu.moneygowhere.commons.core.LogsCenter;
+
+/**
+ * Tab containing the spending graph.
+ */
+public class GraphPanel extends UiPart<Region> {
+    private static final String FXML = "PlaceholderPanel.fxml";
+    private final Logger logger = LogsCenter.getLogger(GraphPanel.class);
```

Comment:
Can remove the loggers if unused in order to pass Codacy test.

Reply:
Done!